src/build/Workleap.DotNet.CodingStandards.props

```diff
@@ -0,0 +1,29 @@
+<Project>
```

Comment:
I've never fully understood why there are 3 folders. But if I well remember using the 3 folders allowed user to consume the package correctly...

Reply:
https://github.com/NuGet/Home/wiki/Allow-package--authors-to-define-build-assets-transitive-behavior

> To construct a package which allows build assets to flow transitively, package author will need to put all these build assets in /buildTransitive as well as /build folder to make the package compatible with packages.config.